packages/cdktf/lib/testing/matchers.ts

```diff
@@ -211,6 +211,30 @@ export function getToHaveResourceWithProperties(
     );
   };
 }
+
+// eslint-disable-next-line jsdoc/require-jsdoc
```

Comment:
Would be super helpful to add the docstrings here and line 221. Allows our documentation generation to improve over time.

Reply:
Sure thing, added them as part of the last commit! :)